plaid/plaid.go

```diff
@@ -7,6 +7,8 @@ import (
 	"io"
 	"io/ioutil"
 	"net/http"
+	
+	"github.com/helloeave/eave-go/config"
```

Comment:
AH! didn't realize this made our public fork refer to our private repo. Hmm, I think there's a fork that already has this change in. I will look for it next week, and maybe we can point to that fork and delete ours.